### PR TITLE
Add postgreql database

### PR DESCRIPTION
**Pull request by sheaj98**

### CODE REVIEW

File: Sources/App/Controllers/TodoController.swift [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import Vapor
+
+/// Controls basic CRUD operations on `Todo`s.
+final class TodoController {
```
Comment (sheaj98) [inline Sources/App/Controllers/TodoController.swift]: Do we need this TodoController? @eeddy

File: Sources/App/Models/Todo.swift [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import FluentSQLite
+import Vapor
+
+/// A single entry of a Todo list.
+final class Todo: SQLiteModel {
```
Comment (sheaj98) [inline Sources/App/Models/Todo.swift]: Same for this file

File: Sources/App/configure.swift [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    services.register(router, as: Router.self)
+
+    // Register middleware
+    var middlewares = MiddlewareConfig() // Create _empty_ middleware config
+    // middlewares.use(FileMiddleware.self) // Serves files from `Public/` directory
+    middlewares.use(ErrorMiddleware.self) // Catches errors and converts to HTTP response
+    services.register(middlewares)
+
+    // Configure a PostgreSQL database
+    let config = PostgreSQLDatabaseConfig(hostname: "localhost", port: 5432, username: "Ethan_Eddy", database: "thekeppochdb", password: nil, transport: .cleartext)
```
Comment (sheaj98) [inline Sources/App/configure.swift]: Should maybe look at different databases based on environment, localhost won't work in digital ocean 😅

File: Sources/App/Controllers/TodoController.swift (hunk 1 above)
Comment (eeddy) [reply Sources/App/Controllers/TodoController.swift]: Removed, git was cuking me

File: Sources/App/Models/Todo.swift (hunk 2 above)
Comment (eeddy) [reply Sources/App/Models/Todo.swift]: Git is a bitch

**Pull request merged**